CONTRIBUTING.md

```diff
@@ -38,27 +38,37 @@ and authorized prior to the PRs inclusion.
 GAP development follows a straightforward branching model.
 
  * Fork our [main development repository](https://github.com/gap-system/gap) on github
- * Clone your fork to your local machine:
+ * Clone your fork to a chosen directory on your local machine using HTTPS (GitHub's recommended cloning protocol):
+```
+$ git clone https://github.com/<your github user name>/gap.git
+```
+This will create a folder called `gap` (in the location where you ran `git clone`) containing the source files, folders and the Git repository.  See [Cloning with HTTPS](https://help.github.com/articles/which-remote-url-should-i-use/#cloning-with-https-recommended) for more information.  You can also clone with plain Git (not recommended as it is insecure) or SSH as well (as [described here](https://help.github.com/articles/which-remote-url-should-i-use/#cloning-with-ssh)).
```

Comment:
I also don't think that we should overload this with alternatives mentioned in brackets. What purpose would this serve?